datastream-server-restli/src/main/java/com/linkedin/datastream/server/dms/DatastreamStore.java

```diff
@@ -55,4 +55,10 @@
    */
   void updatePartitionAssignments(String key, Datastream datastream, HostTargetAssignment hostTargetAssignment,
       boolean notifyLeader) throws DatastreamException;
+
+  /**
+   * Delete the numTasks znode of the datastream associated with the provided key
+   * @param key datastream name of the original datastream whose numTasks znode is to be deleted
```

Comment:
Nit: Reword to
`Name of the original datastream whose numTasks znode is to be deleted`

Reply:
done